app/models/availability.rb

```diff
@@ -39,4 +41,15 @@ def self.process_data
         { 'Date' => date, 'Count' => count }
       end
   end
+
+  private
+
+  def cancel_duplicates
+    previous_availabilities = person.availabilities.for_time_span(start_time..end_time).order(:created_at)
+    previous_availabilities.each do |a|
+      if start_time == a.start_time && end_time == a.end_time
+        a.update(status: "Cancelled") unless self == a
```

Comment:
Sorry, just saw the linked issue suggests the other approach. I'll take a look to see if I can shed any light on the travis failure